src/unix/linux_like/android/b64/aarch64/mod.rs

```diff
@@ -55,6 +55,12 @@ s! {
         pub pc: u64,
         pub pstate: u64,
     }
+
+    pub struct user_fpsimd_struct {
```

Comment:
Can you add this for Linux as well? It's the same struct, it's defined in the kernel headers.